design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -0,0 +1,261 @@
+## Gift Card product type support will be added after the Product Implementation is developed
+type Query {
+    "The requisitionList query returns the contents of customer's  Requisition List"
```

Comment:
Please use `@doc(description: "")` format for documenting the fields and types. 
Is there a benefit of using current approach, will the description be rendered properly in tools like GraphiQL?

Reply:
> Is there a benefit of using current approach

It's the way documentation is spec'd to be written in the IDL:

![image](https://user-images.githubusercontent.com/5233399/82478022-e0c47f00-9a95-11ea-9a95-5573b57ef360.png)

http://spec.graphql.org/June2018/#sec-Descriptions

Reply:
Unfortunately this syntax is not supported by our PHP GraphQL Server and for now we must use `@doc(description: "")` syntax. 
@DrewML feel free to create a ticket to implement support for `" "` and `""" """` syntax. We can then discuss and prioritize it.

@dani97 please change the doc syntax to the supported format.

Reply:
@paliarush updated to `@doc`  syntax. :+1: